canonical-web-and-design/vanillaframework.io
# Remove coding standards page

## Done
- Removal of coding standards page
- Removal of coding standards from the nav
- #winning

## QA
- Pull code
- `./run`
- Visit http://0.0.0.0:8014/
- Notice _coding standards_ is no longer there

## Screenshot
<img width="1440" alt="Screenshot 2019-09-19 at 14 41 00" src="https://user-images.githubusercontent.com/17748020/65244751-85e41d80-daeb-11e9-9631-7ccb5ec4beb5.png">
